src/DacpacTool/PackageBuilder.cs

```diff
@@ -305,7 +306,8 @@ private void WritePart(FileInfo file, Package package, string path)
 
             using (var stream = part.GetStream())
             {
-                var buffer = Encoding.UTF8.GetBytes(File.ReadAllText(file.FullName, Encoding.UTF8) + Environment.NewLine + "GO" + Environment.NewLine);
+                var parser = new ScriptParser(file.FullName, new IncludeVariableResolver());
+                var buffer = Encoding.UTF8.GetBytes(parser.GenerateScript());
```

Comment:
Use the `ScriptParser` to generate the full script after merging any includes.

Reply:
That makes total sense to me :).